var/spack/repos/builtin/packages/conquest/package.py

```diff
@@ -63,12 +63,29 @@ def edit(self, spec, prefix):
 
         lapack_ld = self.spec["lapack"].libs.ld_flags
         blas_ld = self.spec["blas"].libs.ld_flags
+        fftw_ld = self.spec["fftw"].libs.ld_flags
+        libxc_ld = self.spec["libxc"].libs.ld_flags
+
+        if self.version <= Version("1.2"):
+            defs_file = FileFilter("./src/system.make")
+        else:
+            # Starting from 1.3 there's automated logic in the Makefile that picks
+            # from a list of possible files for system/compiler-specific definitions.
+            # This is useful for manual builds, but since the spack will do its own
+            # automation of compiler-specific flags, we need to override it.
+            defs_file = FileFilter("./src/system/system.example.make")
+            makefile = FileFilter("./src/Makefile")
+            makefile.filter("SYSTEM =.*", "SYSTEM = example")
```

Comment:
You can also call `make` with an argument, like in https://github.com/OrderN/CONQUEST-release/blob/00329bba12b4621051d64d87d0232bda63e81443/.github/workflows/makefile.yml#L65 , if it's easier.